src/index.js

```diff
@@ -47,6 +62,18 @@ class LndEngine {
       throw new Error(`${symbol} is not a valid symbol for this engine.`)
     }
 
+    // Expose config publicly that we expect to be used by consumers
+    PUBLIC_CONFIG.forEach((configKey) => {
```

Comment:
Im cool with this, but what was the reasoning for choosing this over simple `get` getters? It'd be more readable just to include only what values you need with `get maxChannelBalance` etc..

Configurations that we need are `secondsPerBlock`, `maxChannelBalance`, `maxPaymentSize`. If the rest of the properties are meant to be private, then we can simply use `currencyConfig` from inside the engine?